FirebaseAuthUI/FUIAuth.h

```diff
@@ -176,6 +176,11 @@ __attribute__((deprecated("Instead use authUI:didSignInWithAuthDataResult:error:
  */
 @property(nonatomic, assign, getter=isSignInWithEmailHidden) BOOL signInWithEmailHidden;
 
+/** @property allowNewEmailAccounts
+ @brief Whether to allow new user sign, defaults to YES.
+ */
+@property(nonatomic, assign, getter=allowedNewEmailAccounts) BOOL allowNewEmailAccounts;
```

Comment:
I don't think this needs a custom getter name (but if you'd like to keep it, consider renaming to `allowsNewEmailAccounts`)